flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
Sadly I cannot support something that I can't test 😟

Reply:
it's clear you are not making the steps to be able to.

Reply:
How do you mean?

Reply:
I moved the rust toolchain to its own module and added two variants, one for x86_64 and one for aarch64.
That being said I do not want to enable aarch64 build until I can get some aarch65 HW to test on.

Reply:
I think this version should be OK from a manifest perspective, please re-open the conversation if not